### PR TITLE
SDL2: update to latest commit

### PR DESCRIPTION
**Pull request by isage**

### CODE REVIEW

File: sdl2/VITABUILD [hunk 1]
```diff
@@ -1,7 +1,7 @@
 pkgname=sdl2
 pkgver=2.0.23
-pkgrel=3
-gitrev=0a1641635d4a378957b077dd45ecfdb63527be26
+pkgrel=4
+gitrev=9670d2bb9ed19e3720c58378fe3eadab1ea0c40e
```
Comment (d3m3vilurr) [inline sdl2/VITABUILD:4]: how about change a version to `2.23.2` and following release of sdl team?

- `cb46e1b` is `2.23.2`
- `9670d2` could be `2.23.2-1`

if you think that we need `9670d2`, change version to `2.23.2-1`
and if not, we can use `prerelease-2.23.2` tag or `cb46e1b`

Comment (isage) [reply sdl2/VITABUILD:4]: We'll switch to new numbering scheme when 2.24 gets released

Comment (d3m3vilurr) [reply sdl2/VITABUILD:4]: they have already changed lib version to 2.23.x https://github.com/libsdl-org/SDL/commit/f1416ef2ba
and now it has a pretty clear meaning that this is a dev/unstable version.  :)
https://github.com/libsdl-org/SDL/issues/5217#issuecomment-1112475879

Comment (isage) [reply sdl2/VITABUILD:4]: I know. But officially new numbering scheme should be used only since 2.24

Comment (d3m3vilurr) [reply sdl2/VITABUILD:4]: huh? so, why did they make a tag to 2.23.2?

Comment (isage) [reply sdl2/VITABUILD:4]: Fine

Comment (d3m3vilurr) [reply sdl2/VITABUILD:4]: thanks. LGTM :+1:

**Review by sharkwouter — APPROVED** [review]

**Pull request merged**